owncloudApp/src/main/java/com/owncloud/android/extensions/AdapterExt.kt

```diff
@@ -0,0 +1,53 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author John Kalimeris
+ * Copyright (C) 2020 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.extensions
+
+import com.owncloud.android.datamodel.OCFile
+import java.util.ArrayList
+import java.util.Vector
+
+fun Vector<OCFile>.filterByQuery(query: String): List<OCFile> {
```

Comment:
Sorry, my bad. This is a vector extension, so this file should be renamed to VectorExt